xcdat/regridder/accessor.py

```diff
@@ -86,53 +97,54 @@ def _get_axis_data(self, name: CFAxisName) -> Tuple[xr.DataArray, xr.DataArray]:
 
         return coord_var, bounds_var
 
-    def horizontal_xesmf(
-        self,
-        data_var: str,
-        output_grid: xr.Dataset,
-        **options: Dict[str, Any],
-    ) -> xr.Dataset:
-        """
-        Wraps the xESMF library providing access to regridding between
-        structured rectilinear and curvilinear grids.
+    if xesmfAvailable:
+        def horizontal_xesmf(
+            self,
+            data_var: str,
+            output_grid: xr.Dataset,
+            **options: Dict[str, Any],
+        ) -> xr.Dataset:
+            """
+            Wraps the xESMF library providing access to regridding between
+            structured rectilinear and curvilinear grids.
 
-        Regrids ``data_var`` in dataset to ``output_grid``.
+            Regrids ``data_var`` in dataset to ``output_grid``.
 
-        Option documentation :py:func:`xcdat.regridder.xesmf.XESMFRegridder`
+            Option documentation :py:func:`xcdat.regridder.xesmf.XESMFRegridder`
 
-        Parameters
-        ----------
-        data_var: str
-            Name of the variable in the `xr.Dataset` to regrid.
-        output_grid : xr.Dataset
-            Dataset containing output grid.
-        options : Dict[str, Any]
-            Dictionary with extra parameters for the regridder.
+            Parameters
+            ----------
+            data_var: str
+                Name of the variable in the `xr.Dataset` to regrid.
+            output_grid : xr.Dataset
+                Dataset containing output grid.
+            options : Dict[str, Any]
+                Dictionary with extra parameters for the regridder.
 
-        Returns
-        -------
-        xr.Dataset
-            With the ``data_var`` variable on the grid defined in ``output_grid``.
+            Returns
+            -------
+            xr.Dataset
+                With the ``data_var`` variable on the grid defined in ``output_grid``.
 
-        Raises
-        ------
-        ValueError
-            If tool is not supported.
+            Raises
+            ------
+            ValueError
+                If tool is not supported.
 
-        Examples
-        --------
+            Examples
+            --------
 
-        Generate output grid:
+            Generate output grid:
 
-        >>> output_grid = xcdat.create_gaussian_grid(32)
+            >>> output_grid = xcdat.create_gaussian_grid(32)
 
-        Regrid data to output grid using regrid2:
+            Regrid data to output grid using regrid2:
 
-        >>> ds.regridder.horizontal_regrid2("ts", output_grid)
-        """
-        regridder = REGRID_TOOLS["xesmf"](self._ds, output_grid, **options)
+            >>> ds.regridder.horizontal_regrid2("ts", output_grid)
```

Comment:
Looks like this wasn't updated `regrid2` -> `xesmf`, a cleanup while this is happening?